### ISSUE TITLE
Page-content template part descends forever in Visual mode

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/fizzie#61]
For the following HTML
```
<!-- wp:group {"backgroundColor":"background","className":"page","layout":{"contentSize":"80%","wideSize":"90%"}} -->
<div class="wp-block-group page has-background-background-color has-background"><!-- wp:post-title {"align":"full","backgroundColor":"light-green-cyan"} /-->

<!-- wp:post-content /--></div>
<!-- /wp:group -->
```

When viewing the **Page content** template part in the visual editor it appears that the `post-title block` is repeated and gets smaller and smaller, until the editor finally gives up with a broken block.
The console log reports: 

Uncaught Error: Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.

Removing the `post-content` block prevents the recursion.
![image](https://user-images.githubusercontent.com/2474435/114054816-16c64980-9888-11eb-8ea4-68701aec2584.png)

![image](https://user-images.githubusercontent.com/2474435/114054932-33fb1800-9888-11eb-8be9-fefe5e4a63af.png)

Gutenberg 10.3.1

**Comment by bobbingwide** [comment]
I reproduced the problem with TT1 blocks

**Comment by bobbingwide** [comment]
I reproduced the problem with the Phoenix theme. 

Using the Site Editor I edited the single page and saved the content group as `single-content`.
Then I tried to edit it from **Appearance > Template parts**

This gif shows the recursion ending with the block failing and the template part's source HTML.
![Fizzie issue 61](https://user-images.githubusercontent.com/2474435/114573883-b0b73900-9c70-11eb-8132-a7878a04b3c0.gif)
